web/src/narrow_state.ts

```diff
@@ -189,7 +196,7 @@ export function pm_emails_string() {
     return operands[0];
 }
 
-export function get_first_unread_info() {
+export function get_first_unread_info(): {flavor: string; msg_id?: number} {
```

Comment:
Oh this can be more specific: `{flavor: "cannot_compute" | "not_found"} | {flavor: "found", msg_id: number}`. This will later avoid an extra branch in `maybe_add_local_messages` when we convert `narrow.js`.

Reply:
Ah cool, great